### PR TITLE
The version of verification for iPXE. 

### PR DESCRIPTION
**Pull request by jsfakian**
It builds a tarball of artifacts to be used for PXE verification

### CODE REVIEW

File: tools/makenet.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@ EVE="$(cd "$(dirname "$0")" && pwd)/../"
 PATH="$EVE/build-tools/bin:$PATH"
 SOURCE="$(cd "$1" && pwd)"
 NET="$(cd "$(dirname "$2")" && pwd)/$(basename "$2")"
+IMG="$3"
```
Comment (rene) [inline tools/makenet.sh:6]: You have introduced a third argument which is mandatory. However, there is no information about it in the help message (line 9) and the checks on line 8 still considers only 2 arguments. Actually I think you could change the order, like this:

`Usage: <input dir> <image file name> <output tar image file>`

So we keep the output tar image file as the last argument, and please, consider use the entire file name as arguemnt, e.g., installer.img, verification.img, etc.

File: Makefile [hunk 2]
```diff
@@ -727,7 +727,7 @@ $(INSTALLER).iso: $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(INSTALLER_IMG) $(CON
 	$(QUIET): $@: Succeeded
 
 $(INSTALLER).net: $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(INSTALLER_IMG) $(CONFIG_IMG) $(PERSIST_IMG) $(KERNEL_IMG) | $(INSTALLER)
-	./tools/makenet.sh $| $@
+	./tools/makenet.sh $| $@ installer
```
Comment (rene) [inline Makefile:730]: Let just `installer` or `verification` as argument it's a bit confusing. I would prefer to use the entire name (intaller.img and verification.img).

**Review by rene — COMMENTED** [review]
@jsfakian , your commit title is too long, it can mess a bit the output of commands like `git log --oneline`. Please, break the information into a short title + commit message (explaining the changes).

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Please address @rene 's comments.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**